### PR TITLE
[demo] HRINT-1320 Fix DatabaseDoesNotExistException

### PR DESCRIPTION
**Pull request by lukaszdobrzynski**

### CODE REVIEW

File: DemoCommon/Utils/Database/DatabaseApi.cs [hunk 1]
```diff
@@ -48,13 +48,20 @@ public Task CreateSampleData(IDocumentStore documentStore)
...
+               await documentStore.Maintenance.Server.SendAsync(operation);
+            }
+            catch (DatabaseDoesNotExistException)
+            {
+                
+            }
         }
 
-        public Task DeleteDatabase(IDocumentStore documentStore) => DeleteDatabase(documentStore, documentStore.Database);
+        public async Task DeleteDatabase(IDocumentStore documentStore) => await DeleteDatabase(documentStore, documentStore.Database);
```
Comment (ppekrol) [inline DemoCommon/Utils/Database/DatabaseApi.cs]: async/await is not needed here

File: DemoServer/Utils/Database/DatabaseSetup.cs [hunk 2]
```diff
@@ -84,10 +84,10 @@ public Task DeleteUserDatabase(Guid userId)
             return _databaseApi.DeleteDatabase(documentStore);
         }
 
-        public Task DeleteMediaDatabase(Guid userId)
+        public async Task DeleteMediaDatabase(Guid userId)
         {
             var documentStore = _mediaStoreCache.GetEntry(userId);
-            return _databaseApi.DeleteDatabase(documentStore);
+            await _databaseApi.DeleteDatabase(documentStore);
```
Comment (ppekrol) [inline DemoServer/Utils/Database/DatabaseSetup.cs]: async/await is not needed here

**Comment by topalach** [comment]
@lukaszdobrzynski please follow Paweł's comments. Besides that, looks good to me..

File: DemoCommon/Utils/Database/DatabaseApi.cs (hunk 1 above)
Comment (lukaszdobrzynski) [reply DemoCommon/Utils/Database/DatabaseApi.cs]: Done.

File: DemoServer/Utils/Database/DatabaseSetup.cs (hunk 2 above)
Comment (lukaszdobrzynski) [reply DemoServer/Utils/Database/DatabaseSetup.cs]: Done.

**Pull request merged**